pkg/consoleplugin/scriptserver/scriptserver_test.go

```diff
@@ -0,0 +1,25 @@
+package scriptserver
+
+import (
+	"github.com/stretchr/testify/require"
+	"io"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestScriptServer(t *testing.T) {
+	s := NewScriptServer()
+
+	req := httptest.NewRequest("GET", "/pendo.ts", nil)
+	resp := httptest.NewRecorder()
+
+	s.HandleScriptRequest(resp, req)
+
+	body, err := io.ReadAll(resp.Body)
+	require.NoError(t, err)
+
+	require.Len(t, body, 1896)
+	require.True(t, strings.HasPrefix(string(body), "// initialize pendo"))
+
+}
```

Comment:
could you please cover also the other files, and some (or all) the error cases? you can do it in for cycle and ideally multiple calls in parallel to verify that the locks and cache work as expected.

Reply:
All non-dynamically generated files are now included in the tests in some form.